manifest.py

```diff
@@ -234,3 +234,70 @@ def create_manifest(sources, tileset, license, account, product, date, notes,
 if __name__ == "__main__":
     create_manifest()
 
+# PyTest only below this line
+
+from click.testing import CliRunner
+from jsonschema import FormatChecker
+from jsonschema import validate
+import pytest
+
+import json
+
+
+@pytest.fixture(scope='function')
+def manifest_args():
+    """Fixture for manifest args recreated on each function call."""
+    return ['-t', 'accountname.tileset',
+            '--license', '"CC BY-SA"',
+            '--account', 'accountname',
+            '--product', 'productname',
+            '--date', '2018'
+            ]
+
+
+def test_create_manifest(manifest_args):
+    """Tests the create manifest function checking for exceptions."""
+    runner = CliRunner()
+    with runner.isolated_filesystem():
+        with open('sources.txt', 'w') as f:
+            f.write('s3://my-bucket/test.tif')
```

Comment:
On the other hand, this sources.txt file is a great fixture candidate.

Reply:
+1